testsuites/CBLTester/System_test_multiple_device/test_system_testing.py

```diff
@@ -0,0 +1,283 @@
+        except:
+            pass
+    # Configure replication with push_pull for all db
+    replicator_obj_list = []
+    replicator_list = []
+    for base_url, cbl_db, query in zip(base_url_list, cbl_db_list, query_obj_list):
+        repl_obj = Replication(base_url)
+        replicator_obj_list.append(repl_obj)
+        session, _, repl = repl_obj.create_session_configure_replicate(
+            base_url, sg_admin_url, sg_db, username, password, channels_sg, sg_client, cbl_db, sg_blip_url, continuous=True)
+        replicator_list.append(repl)
+        query.query_get_docs_limit_offset(cbl_db, limit=query_limit, offset=query_offset)
+
+    current_time = datetime.now()
+    running_time = current_time + timedelta(minutes=up_time)
+
+    _check_doc_count(db_obj_list, cbl_db_list)
+    x = 1
+    while(running_time - current_time > timedelta(0)):
+
+        print '*' * 20
+        print "Starting iteration no. {} of system testing".format(x)
+        print '*' * 20
+        x += 1
+        ######################################
+        # Checking for doc update on SG side #
+        ######################################
+        docs_to_update = random.sample(doc_ids, num_of_docs_to_update)
+        sg_docs = sg_client.get_all_docs(url=sg_url, db=sg_db, auth=session)["rows"]
+        sg_docs = [doc for doc in sg_docs if doc["id"] in docs_to_update]
+        print "Updating {} docs on SG - {}".format(len(docs_to_update),
+                                                   docs_to_update)
+        sg_client.update_docs(url=sg_url, db=sg_db, docs=sg_docs,
+                              number_updates=num_of_updates, auth=session, channels=channels_sg)
+
+        # Waiting until replicator finishes on all dbs
+        for repl_obj, repl, cbl_db, query in zip(replicator_obj_list,
+                                                 replicator_list,
+                                                 cbl_db_list,
+                                                 query_obj_list):
+            t = Thread(target=_replicaton_status_check, args=(repl_obj, repl))
+            t.start()
+            t.join()
+            query.query_get_docs_limit_offset(cbl_db, limit=query_limit, offset=query_offset)
+
+        #######################################
+        # Checking for doc update on CBL side #
+        #######################################
+        docs_to_update = random.sample(doc_ids, num_of_docs_to_update)
+        i = 0
+        for db_obj, cbl_db, repl_obj, repl, query in zip(db_obj_list,
+                                                         cbl_db_list,
+                                                         replicator_obj_list,
+                                                         replicator_list,
+                                                         query_obj_list):
+            updates_per_db = len(docs_to_update) / len(db_obj_list)
+            print "Updating {} docs on {} db - {}".format(updates_per_db,
```

Comment:
can you either remove print statements or replace with log_info?

Reply:
changed it to log_info